oaknational/Oak-Web-Application
# BWBWS - Added reload to mux player when error triggered

## Description

Music year: 1994

I've added a reload to the video component for when an error is triggered within mux.

There is a temporary button to manually trigger an error and reload the component up to 3 times. I'll remove this before a final review.

Just want to get some opinions on this first. I also thing we could reload the signed url, but that is another ticket I think.

## How to test

1. Go to https://deploy-preview-2581--oak-web-application.netlify.thenational.academy/pupils/programmes/drama-primary-year-3-l/units/stories-shared-across-the-world-2d56/lessons/dragons-c8v3cc/video
2. Play the video and manually cause an error by clicking the trigger error button, the video component should reload and start playing from the same location.

![mux reload](https://github.com/oaknational/Oak-Web-Application/assets/587241/e9330fb9-1894-473d-bdb7-9839bc49ea34)


## Checklist
